This patch gives `runtest' a `$0' that contains `runtest' instead
of `.runtest-wrapped'.  This is required because `runtest' checks
what `$0' looks like to determine the target.

--- dejagnu-1.4.4/runtest	2003-08-17 03:27:59.000000000 +0200
+++ dejagnu-1.4.4/runtest	2010-03-18 15:57:58.000000000 +0100
@@ -10,6 +10,7 @@
 # Get the execution path to this script and the current directory.
 #
 mypath=${0-.}
+mypath="`echo $mypath | sed -'es|\.runtest-wrapped|runtest|g'`"
 if expr ${mypath} : '.*/.*' > /dev/null
 then
     :

